### PR TITLE
LOG-2906: Add ability forward logs to Splunk output

### PR DESCRIPTION
**Pull request by jcantrill**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>
Attempt to replace #1605 that is for some reason having test issues

cc @vparfonov

### CODE REVIEW

File: olm_deploy/scripts/env.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -eou pipefail
 
 LOGGING_VERSION=${LOGGING_VERSION:-5.6}
 LOGGING_FLUENTD_VERSION=${LOGGING_FLUENTD_VERSION:-1.14.6}
-LOGGING_VECTOR_VERSION=${LOGGING_VECTOR_VERSION:-0.21-rh}
+LOGGING_VECTOR_VERSION=${LOGGING_VECTOR_VERSION:-v0.23-rh}
```
Comment (vimalk78) [inline olm_deploy/scripts/env.sh:6]: change this back to v0.21-rh

Comment (vparfonov) [reply olm_deploy/scripts/env.sh]: without `v` to 0.21-rh, right @vimalk78 ?

Comment (vimalk78) [reply olm_deploy/scripts/env.sh]: yes, without `v`

**Comment by vparfonov** [comment]
need to squash commits

**Pull request merged**